### ISSUE TITLE
[BUG] Finance totals for parking not matching controllers site

### ISSUE THREAD

**Issue by Charlie-Henry** [cityofaustin/atd-data-tech#11578]
Had a meeting with Erin W. and DeMarcos about fixing the parking dashboard to show the correct amount that matches what was deposited shown on the controller's website. It looks like the totals themselves are not correct but the total over several days will end up being correct. So it looks like I'm not grouping the dates correctly anymore.

I think this is possibly related to Fiserv changing their client line site:
https://github.com/cityofaustin/atd-data-tech/issues/10944

Could also be on Flowbird's end.

Putting a 3 on this preemptively as it feels like this might be a bigger problem than just messing with the dates in Power BI until we get some that work.

**Comment by Charlie-Henry** [comment]
Did some investigation on this bug this morning. Notes on what I tried:

- Found the correct page on the new Fiserv website for downloading the reports we need. It's on the left side of the page under Settlement > Search (at the bottom).
- Using DeMarcos's spreadsheet I was not finding a pattern of why certain totals weren't matching up. I tried looking at just account numbers, devices, and by card type. This [was an issue previously](https://github.com/cityofaustin/atd-data-tech/issues/8813) where we had one type of transaction that has to be manually reduced by one day in order to match up with what the controller's page is saying.
- Testing downloading a new report from Fiserv shows the same `batch_date` as we have as `submit_date`, so same problem.
- It looks like `funded_date` _sometimes_ matches up with what's in the controllers page for MCVD (AKA not AMEX cards) for account #855. No idea why it works then stops for a period of time.
